tests/credentials_tests.rb

```diff
@@ -1,6 +1,31 @@
 # frozen_string_literal: true
 
 Shindo.tests('AWS | credentials', ['aws']) do
+  unchecked_constants = [:CredentialFetcher, :VERSION, :Errors, :Mock, :ServiceMapper, :Federation, :STS, :SignatureV4, :EFS, :Parsers]
```

Comment:
`Parsers` is not inherited Fog::Service

https://github.com/fog/fog-aws/blob/v3.19.0/lib/fog/aws/parsers/lambda/base.rb#L1-L3